setup.cfg

```diff
@@ -43,3 +43,11 @@ ignore =
     F403
     E302
     F541
+
+
+[isort]
```

Comment:
@glenn-jocher here is the configuration :]
feel free to edit it how you wish and let the bot update the PR

Reply:
@Borda awesome!! Thanks I'll experiment with it.